api-report/runtime-definitions.api.md

```diff
@@ -154,6 +164,8 @@ export interface IFluidDataStoreContext extends IEventProvider<IFluidDataStoreCo
     readonly attachState: AttachState;
     // (undocumented)
     readonly baseSnapshot: ISnapshotTree | undefined;
+    // @deprecated (undocumented)
+    bindToContext(): void;
```

Comment:
maybe make this optional, that will make it non-breaking if people don't care about his api

Reply:
I may just leave it off here altogether and have FluidDataStoreRuntime.bindToContext call this.dataStoreContext.makeLocallyVisible() directly rather than having the bindToContext shim on DataStoreContext.  Will make this PR smaller, but less of a direct revert.

Reply:
Leaving it on for now, it really needs to be there for the code I'm adding back to work.  I'll refactor it away in next once these changes get there.